### PR TITLE
Cleaned up file structure and enforced good Shell practices

### PR DESCRIPTION
**Pull request by Wolveix**
- Added superuser check
- Changed echo usage to printf for better control
- Improved package manager usage
- Wrapped variables in quotation marks to prevent globbing/word splitting

### CODE REVIEW

**Comment by ScottEnock** [comment]
Thanks for the commits! They're unbelievably helpful. I'm going to pull the changes and give them a test. 😃

File: mongo-setup.sh [hunk 1]
```diff
@@ -1,51 +1,48 @@
-
 #!/usr/bin/env bash
 
-echo "MongoDB Startup Running..."
+printf "\\nMongoDB Startup Running...\\n"
+if [ "$EUID" -ne 0 ]; then
```
Comment (ScottEnock) [inline mongo-setup.sh]: I'm encountering an error when running the script on Ubuntu & Debian:
MongoDB Startup Running...
mongo.sh: 4: [: Illegal number:

**Comment by Wolveix** [comment]
I can't reproduce this, the script is working fine for me. Are you sure there isn't an issue in your environment?

**Comment by ScottEnock** [comment]
Alas! I was executing the script with the sh command instead of the bash command which doesn't have as many features as bash. I'll merge now.

**Pull request merged**